fdbserver/Resolver.actor.cpp

```diff
@@ -423,6 +428,20 @@ ACTOR Future<Void> resolveBatch(Reference<Resolver> self, ResolveTransactionBatc
 						writtenTLogs.insert(i);
 					}
 				} else {
```

Comment:
Here is another issue that I noticed here (discussed this with Dan): My understanding is that we currently never get into this "else" branch as the if-condition above always returns true (because "reply.privateMutationCount", which is essentially the value of "LogPushData::subsequence", never goes to zero; it starts from one).